modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -3,26 +3,50 @@
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
 :_content-type: PROCEDURE
-[id="configuring-host-network-interfaces-in-the-install-config-yaml-file_{context}"]
-= (Optional) Configuring host network interfaces
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= Optional: Configuring host network interfaces in the `install-config.yaml` file
 
-During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration.
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState.
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Host network interface configuration is set during the installation and provisioning process and includes advanced options, such as setting static IPs per host. Before configuring host network interfaces, review the {product-version} 4.10 release notes. 
 ====
 
-See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
+* You configured a `PTR` DNS record with a valid hostname for each node with a static IP address.
+
+* You installed the `jq` package.
+
+.Procedure
+
+Optional: Consider testing the `networkConfig` YAML syntax before including it in the `install-config.yaml` file. The installation program will not check the `NMState` YAML syntax.
+
+.. Save the valid `networkConfig` YAML syntax to a file.
+.. Run the following command:
++
 .Example
```

Comment:
You don't need titles for command blocks. Use `+` to align the command with the step block.

```suggestion
+
```